app/components/Category.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component, PropTypes } from 'react';
+import { Link } from 'react-router';
+
+var icons = require.context("../assets/images", true, /ic-.*\.png$/i);
```

Comment:
This looks hairy, but it lets us leverage webpack when using these images even though we've got the indirection that keeps us from referring to them using constants in the code (that is, the server returns the image name).